src/edu/csus/ecs/pc2/core/imports/clics/CLICSEventType.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+/**
+ * CLICS Event Feed Event Type.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public enum CLICSEventType {
+
+    /**
+     * accounts event
+     */
+    ACCOUNTS("accounts"),
+    /**
+     * clarifications event
+     */
+    CLARIFICATIONS("clarifications"),
+    /**
+     * contest event
+     */
+    CONTEST("contest"),
+    /**
+     * groups event
+     */
+    GROUPS("groups"),
+    /**
+     * judgement-types event
+     */
+    JUDGEMENT_TYPES("judgement-types"),
+    /**
+     * judgements event
+     */
+    JUDGEMENTS("judgements"),
+    /**
+     * languages event
+     */
+    LANGUAGES("languages"),
+    /**
+     * organizations event
+     */
+    ORGANIZATIONS("organizations"),
+    /**
+     * persons event
+     */
+    PERSONS("persons"),
+    /**
+     * problems event
+     */
+    PROBLEMS("problems"),
+    /**
+     * runs event
+     */
+    RUNS("runs"),
+    /**
+     * state event
+     */
+    STATE("state"),
+    /**
+     * submissions event
+     */
+    SUBMISSIONS("submissions"),
+    /**
+     * teams event
+     */
+    TEAMS("teams");
+
```

Comment:
The above list of CLICS events type is missing the `awards` event type defined in the CLICS 2202-07 specification.  Since we are trying to bring PC2 code up to  compliance with the spec, I think this class should include all event types which are defined in the spec.

Reply:
pushed fix, 3 days ago.